src/utils.js

```diff
@@ -34,3 +34,33 @@ module.exports.reflect = async function (thenable, args) {
     return error;
   }
 };
+
+/**
+ * Creates promise that will resolve after the given time (in ms).
+ */
+
+module.exports.timeout = function (ms, returnValue) {
+  return new Promise(resolve => setTimeout(() => resolve(returnValue), ms));
+};
+
+/**
+ * Validate logger.
+ */
+
+module.exports.validateLogger = function (logger) {
+  if (typeof logger !== 'object') {
+    throw new Error(`Logger instance is invalid`);
+  }
+  const requiredLogMethods = ['info', 'warn', 'error'];
+
+  for (const logMethod of requiredLogMethods) {
+    if (!logger?.[logMethod]) {
+      throw new Error(`Logger instance is missing required log method '${logMethod}'`);
```

Comment:
```suggestion
      throw new Error(`Logger instance is missing required log method \`${logMethod}\``);
```

Reply:
Damn, is that in any standard we use? Because that does not look good. I would rather take out the quotes, there are other places where we are not using them and arguably could.

Reply:
It is the standard, indeed.
https://github.com/uphold/backend/blob/master/test/src/authentication/managers/authentication-approval-manager_test.js#L221

Reply:
That tells me nothing. First, it is not a standard, second, it is not an error, third it is not a string template.

Searching for more comparable examples, I found 2, but both are different, so I'm guessing there is no standard:
https://github.com/uphold/backend/blob/11c62d2000134f16f25dfd53bf2c9fe785a1776e/src/exchange/gateways/songbird-gateway-provider.js#L344
https://github.com/uphold/backend/blob/d5f05622115e9478048eca395fa5f0e4258836cc/src/exchange/tickers/abstract-ticker-provider.js#L40

Reply:
Well, that was the standard that was taught to me.
Now, if everyone throws in their own standard, as usual, it fails to longer be a standard. 🤷

Personally, I don't care which "standard" is applied, as I'm more concerned with consistency, so I'll leave it to be debate by colleagues with stronger opinions on this topic.

Reply:
This is something that can easily be added to the eslint config, so if it isn't there yet, I'm not sure a "standard" has been decided yet. :laughing:

Reply:
For reference, regarding `That tells me nothing. First, it is not a standard, second, it is not an error, third it is not a string template.`:

Regardless on whether it is a string template or a static string, I believe it would be good common sense to examine the output. If the output of a log is commonly:
'This is a message that references method `foobar()`'

I don't think it really matters what's implemented underneath. My 2 cents.

Reply:
```
> Error: Logger instance is missing required log method 'info'
> Error: Logger instance is missing required log method `info`
```

Either looks fine to me, we could also go with `"`.